rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,228 @@
+import logging
+from collections import namedtuple
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import gettext_lazy as _
+
+from rdmo.conditions.models import Condition
+from rdmo.domain.models import Attribute
+from rdmo.management.utils import replace_uri_in_template_string
+from rdmo.projects.models import Value
+from rdmo.questions.models import Page, Question, QuestionSet
+from rdmo.tasks.models import Task
+from rdmo.views.models import View
+
+logger = logging.getLogger(__name__)
+
+
+ModelHelper = namedtuple('ModelHelper', ['model', 'lookup_field', 'field_introspection', 'replacement_function'])
+
+REPLACE_ATTRIBUTE_MODEL_HELPERS = [
+    ModelHelper(Page, 'attribute', False, None),
+    ModelHelper(QuestionSet, 'attribute', False, None),
+    ModelHelper(Question, 'attribute', False, None),
+    ModelHelper(Condition, 'source', False, None),
+    ModelHelper(Task, 'start_attribute', False, None),
+    ModelHelper(Task, 'end_attribute', False, None),
+    ModelHelper(Value, 'attribute', False, None),
+    ModelHelper(View, 'template', True, replace_uri_in_template_string)
+]
+
+
+def get_queryset(model, lookup_field, field_introspection, *_, attribute=None):
+    if attribute is None:
```

Comment:
This can be skipped if the attribute would not be a kwarg. Also validation should happen at the beginning of the script.

Reply:
thanks, moved all validation to the start